docs/whitepaper/federated_sidetree.md

```diff
@@ -194,22 +178,75 @@ It is necessary for a logical Hub to have tighter coupling than the interconnect
 
 The multiple organization Hub case is an evolution of the TrustBloc DID, where consortium policy and discovery is defined on top of the [Sidetree REST API](https://identity.foundation/sidetree/api/). The Hub case in this document adds description for enabling multiple external trees to be observed.
 
+## Sidetree anchor encoding
+
+This DID method makes use of two Merkle Tree structures. The first structure provides the graph of how Batch Writers have created Anchors built upon previous anchors. The second structure provides a log of each anchor object. There can be multiple logs that contain each anchor object.
+
+The Merkle Tree anchor graph is stored as links within the CAS objects, and each object has a CID. Each CAS Anchor object contains:
+
+* An anchor string.
+* Proof from each Merkle Tree log where the Batch Writer explicitly submitted the anchor object.
+* Reference to the prior anchor for each unique DID suffix included in the operations referenced by the anchor.
+
+With these CAS anchor objects, a graph can be formed of the anchor relationships from the viewpoint of the batch writer. By having a CAS-based graph, observers are able to apply operations to a suffix if they become aware of a new anchor via mechanisms such as gossip. The following diagram illustrates the structure.
+
+```mermaid
+graph TD
+    NEW_ANCHOR[New Anchor] --- PARENT1[Parent Anchor 1]
+    NEW_ANCHOR --- PARENT2[Parent Anchor 2]
+
+    NEW_ANCHOR --- PARENT3[Parent Anchor 3]
+    PARENT1 --- GRANDPARENT1[Grandparent Anchor 4]
+    PARENT1 --- GRANDPARENT2
+    PARENT2 --- GRANDPARENT2[Grandparent Anchor 5]
+    PARENT2 --- GRANDPARENT3[Grandparent Anchor 6]
+    PARENT3 --- GRANDPARENT4[Grandparent Anchor 7]
+```
+
+The second structure -- the Merkle Tree log -- is created independently of the CAS. Each Merkle Tree log provides an independent witness that an anchor has been published at a certain time. By monitoring these logs, observers are able to discover the history of published anchors. The history is useful for several reason:
+
+* As it is possible for different hash chain for a suffix to be created into the CAS (by a DID controller), a log can be used to disambiguate.
```

Comment:
Can you provide an example for "as it is possible for different hash chain for a suffix to be created into the CAS (by a DID controller)"?

Reply:
@sandrask This can happen when the DID controller submits the different operations (with the same reveal value) to batch writers that share a common CAS.